src/plugins/plugins.ts

```diff
@@ -79,6 +80,16 @@ const PluginConfigs: RendererPluginType[] = [
         mimeTypes: ["image/*"],
         simpleComponent: SimpleImageRenderer,
     },
+    {
```

Comment:
How does this look so far?

I am not exactly sure the difference between simple and full plugins, but I assume my plugin is simple because there probably won't be any packet streaming and it's probably less complex than codeedit

Reply:
never mind, currently using full plugin